docs/reference/api/remote_api_client_libraries.md

```diff
@@ -203,6 +203,12 @@ will add the libraries here.
     </tr>
     <tr>
       <td>Rust</td>
+      <td>shiplift</td>
+      <td><a class="reference external" href="https://github.com/softprops/shiplift>https://github.com/softprops/shiplift</a></td>
```

Comment:
You forgot a closing quote here (after `href="https://github.com/softprops/shiplift`)